### PR TITLE
Adds Rocky 8 repository config for local pulp

### PR DESCRIPTION
**Pull request by jovial**
We support Rocky 8 as a host OS with os_distribution == 'rocky'. This adds the configuration necessary to use the RPM snapshots from ark.

### CODE REVIEW

**Review by markgoddard — COMMENTED** [review]
Looks like a good start. You'll need some changes in stackhpc.yml and dnf.yml to use the new repos.

File: etc/kayobe/pulp.yml [hunk 1]
```diff
@@ -164,6 +164,21 @@ stackhpc_pulp_distribution_deb_production:
...
+stackhpc_pulp_sync_rocky_8: "{{ os_distribution == 'rocky' }}"
+# Rocky 8 minor version number. Supported values: 6.
+stackhpc_pulp_sync_rocky_8_minor_version: 6
+# Rocky 8 Snapshot versions. The defaults use the appropriate version from
+# pulp-repo-versions.yml for the selected minor release.
+stackhpc_pulp_sync_rocky_8_appstream_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_appstream_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_baseos_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_baseos_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_extras_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_extras_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_nfv_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_nfv_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_powertools_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_powertools_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
```
Comment (markgoddard) [inline etc/kayobe/pulp.yml]: minor nit: I'd have gone with `s/sync/repo/` here - the sync variables are flags

Comment (jovial) [reply etc/kayobe/pulp.yml]: Gone with your suggestion here, thanks.

**Review by markgoddard — APPROVED** [review]

**Review by markgoddard — APPROVED** [review]

**Comment by jovial** [comment]
Sorry for all these updates. The merge button was greyed out. Doesn't explicitly say why, but I guess its because of "This branch is out-of-date with the base branch".

**Comment by bbezak** [comment]
are you planning to add dnf.yml changes in this PR as well? @jovial

**Comment by jovial** [comment]
> are you planning to add dnf.yml changes in this PR as well? @jovial

Yeah, I've done those here:

```
https://github.com/stackhpc/stackhpc-kayobe-config/pull/152
```

**Pull request merged**